README.md

```diff
@@ -88,15 +88,23 @@ exited cleanly
 
 AppBase maintains a singleton `application` instance which can be accessed via `appbase::app()`.  This 
 application owns a `boost::asio::io_service` which starts running when `appbase::exec()` is called. If 
-a plugin needs to perform IO or other asynchronous operations then it should dispatch it via 
-`app().get_io_service().post( lambda )`.  
+a plugin needs to perform IO or other asynchronous operations then it should dispatch it via `application`
+`io_service` which is setup to use an execution priority queue.
+```
+app().post( appbase::priority::low, lambda )
+```
+OR
+```
+delay_timer->async_wait( app().get_priority_queue().wrap( priority::low, lambda ) );
+```
+Use of `get_io_service()` directly is not recommended as the priority queue will not be respected. 
```

Comment:
Perhaps the last 10 usages of `get_io_service()` should be refactored and the method eliminated?

Reply:
Yes, that would be nice. Feel free.